src/main/java/lotto/io/ConsoleOutputHandler.java

```diff
@@ -80,6 +85,9 @@ private void printExceptRankNone(Rank rank, Integer count) {
     }
 
     private String getResultForRow(Rank rank, int matchedLottoCount) {
+        if (rank == Rank.SECOND) {
```

Comment:
객체에게 메시지를 보내서 처리해보면 어떨까요?
```suggestion
        if (rank.isSecond()) {
```

Reply:
네 감사합니다.